block/pod.yaml

```diff
@@ -14,4 +14,5 @@ spec:
   volumes:
     - name: data
       persistentVolumeClaim:
-        claimName: block-claim
\ No newline at end of file
+        claimName: block-claim
+
```

Comment:
not exactly a empty new line, you can refer https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log to understand what I'm talking about

Reply:
fixed